src/pickWithDefault.js

```diff
@@ -0,0 +1,36 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * Similar to `pick` except that this one includes a `key: <default>` pair for
+ * properties that don't exist
+ *
+ * **Note:** A property set to undefined will be left untouched.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.22.2
+ * @category Object
+ * @sig [k] -> * -> {k: v} -> {k: v}
```

Comment:
@davidchambers - Is there an example signature I can follow?  I don't see 'Array String' anywhere but i do see ['[String]'](http://ramdajs.com/0.22.1/docs/#mapObjIndexed).  I was about to copy paste exactly what you put until I checked the website.